tests/test-dirs/signature-help/issue_let_in_binding.t

```diff
@@ -0,0 +1,50 @@
+  $ cat > test1.ml <<'EOF'
+  > let _ =
+  > let f = List.map   in
+  > let _ = 5 in
+  > 7
+  > EOF
+
+  $ $MERLIN single signature-help -position 2:17 -filename test < test1.ml
+  {
+    "class": "return",
+    "value": {
+      "signatures": [
+        {
+          "label": "List.map : ('a -> 'b) -> 'a list -> 'b list",
+          "parameters": [
+            {
+              "label": [
+                11,
+                21
+              ]
+            },
+            {
+              "label": [
+                25,
+                32
+              ]
+            }
+          ]
+        }
+      ],
+      "activeParameter": 0,
+      "activeSignature": 0
+    },
+    "notifications": []
+  }
```

Comment:
We often try to reduce the output by filtering with `jq`. 
For example you can use `jq '.value.signatures[0].label'` here, since we are only interested in the fact that there is a result.  `jq '.value.signatures[0]` is also a reasonable choice to also check that result.